examples/MyModule/configure.ac

```diff
@@ -0,0 +1,190 @@
+AC_PREREQ(2.52)
```

Comment:
Yes, we should absolutely. To keep things simple, though, I fixed in this PR only those things that absolutely needed fixing to make things work again and solve Janroel's problem. I thought to leave the remaining necessary fixes to this file here until PR #7 is accepted. We should then update this file to libtool initialisation etc as in the newest version of the main configure.ac.